src/components/Profile/ProfileLink/ProfileLink.css

```diff
@@ -12,4 +12,5 @@
 .profile__header__link a {
   text-decoration: none;
   color: inherit;
+  display: contents;
```

Comment:
This solution seems a bit hacky, and the gap between the icon and the text is not clickable.